emscripten/ARToolKitJS.cpp

```diff
@@ -821,23 +821,6 @@ extern "C" {
 		return -1;
 	}
 
-	/*
-	 * Transfer data
-	 */
-
-	void transferDataToHeap(int id, ARUint8  *data) {
```

Comment:
I m not sure but it seems that is not correct, i should check and maybe i will restore it.